### PR TITLE
turn on playlist validation

### PR DESCRIPTION
**Pull request by OscarQQ**
Signed-off-by: Yixin Qian <Yixin.Qian@ibm.com>

### CODE REVIEW

**Comment by OscarQQ** [comment]
x86-64_windows,ppc64_aix,x86-64_linux jdk 8/11  Grinder_TKG/1235/

**Review by llxia — APPROVED** [review]
LGTM

**Comment by karianna** [comment]
@OscarQQ Can this be taken out of draft?

File: src/org/testKitGen/PlaylistInfoParser.java [hunk 1]
```diff
@@ -79,7 +79,7 @@ private void validate() {
 				System.err.println("Column: "+e.getColumnNumber());
 				System.err.println("Message: "+e.getMessage());
 				//TODO: turn on validation after playlist changes
```
Comment (renfeiw) [inline src/org/testKitGen/PlaylistInfoParser.java:81]: Please remove this comment as it's not needed.

**Review by renfeiw — APPROVED** [review]
LGTM

**Pull request merged**